sgp4/export.py

```diff
@@ -0,0 +1,101 @@
+"""
+Export orbit data to a Two-Line-Element representation.
+"""
+from sgp4.io import compute_checksum
+
+from sgp4.model import Satrec
+
+
+def export_tle(satrec):
+    """
+     Generate the TLE for the data in the `Satrec` object.
+
+    Forms the two string lines of the TLE and returns them as a Tuple.
+
+    Parameters
+    ----------
+    satrec : Satrec
+        Object that holds the orbit information
+
+    Returns
+    -------
+    (line1, line2) : (str, str)
+    """
+
+    # Define constants
+    from math import pi
```

Comment:
Typically in Python imports are moved to the top-level, so the import statement doesn't have to run over again every time the function is called. Try moving it out to the module level, and try reading PEP-8 which has good ideas about how to format imports. Thanks!

Reply:
Argh, I must have chosen the "import locally" option in PyCharm by mistake. To be fixed right away.